Modules/IO/JPEG/test/itkJPEGImageIOCMYKTest.cxx

```diff
@@ -46,31 +46,24 @@ itkJPEGImageIOCMYKTest(int argc, char * argv[])
 
     ITK_TRY_EXPECT_NO_EXCEPTION(reader->Update());
 
-    if (!(io->GetPixelType() == itk::CommonEnums::IOPixel::RGB))
-    {
-      std::cout << "Expected RGB. Test failed" << std::endl;
-      return EXIT_FAILURE;
-    }
+    ITK_TEST_EXPECT_TRUE(!(io->GetPixelType() == itk::CommonEnums::IOPixel::RGB));
   }
 
   {
     itk::JPEGImageIO::Pointer io = itk::JPEGImageIO::New();
 
     itk::ImageFileReader<ImageType>::Pointer reader = itk::ImageFileReader<ImageType>::New();
 
-    io->SetCMYKtoRGB(false);
+    auto cmykToRGB = false;
+    ITK_TEST_SET_GET_BOOLEAN(io, CMYKtoRGB, cmykToRGB);
 
     reader->SetFileName(argv[1]);
 
     reader->SetImageIO(io);
 
     ITK_TRY_EXPECT_NO_EXCEPTION(reader->Update());
 
-    if (!(io->GetPixelType() == itk::CommonEnums::IOPixel::VECTOR))
-    {
-      std::cout << "Expected VECTOR. Test failed" << std::endl;
-      return EXIT_FAILURE;
-    }
+    ITK_TEST_EXPECT_TRUE(!(io->GetPixelType() == itk::CommonEnums::IOPixel::VECTOR));
```

Comment:
Wrong
ITK_TEST_EXPECT_TRUE(io->GetPixelType() == itk::CommonEnums::IOPixel::VECTOR);

Reply:
:+1: Thanks for catching this @issakomi. Done in d09c797.

Reply:
OK. IMHO, that macro does exactly the same, i hope, but with much more code, useless `std::cerr`s and flushes, i don't understand that is the advantage. Even the explanation _"Expected VECTOR. Test failed"_ was not bad, anyway it will be required to search for the line in code, etc. I shall approve, no need to answer, i know about SW guide guidelines.

```cpp
#define ITK_TEST_EXPECT_TRUE(command)                                           \
  {                                                                             \
    CLANG_PRAGMA_PUSH                                                           \
    CLANG_SUPPRESS_Wfloat_equal bool _ITK_TEST_EXPECT_TRUE_command(command);    \
    CLANG_PRAGMA_POP                                                            \
    if (!(_ITK_TEST_EXPECT_TRUE_command))                                       \
    {                                                                           \
      std::cerr << "Error in " << #command << std::endl;                        \
      std::cerr << "  In " __FILE__ ", line " << __LINE__ << std::endl;         \
      std::cerr << "Expected true" << std::endl;                                \
      std::cerr << "  but got  " << _ITK_TEST_EXPECT_TRUE_command << std::endl; \
      return EXIT_FAILURE;                                                      \
    }                                                                           \
  }                                                                             \
  ITK_MACROEND_NOOP_STATEMENT
```